Doc/library/faulthandler.rst

```diff
@@ -66,10 +66,24 @@ Dumping the traceback
       Added support for passing file descriptor to this function.
 
 
+Dumping the C stack
+-------------------
+
+.. versionadded:: next
+
+.. function:: dump_c_stack(file=sys.stderr)
+
+   Dump the C stack trace of the current thread into *file*.
+
+   This is done with `GNU backtraces <https://www.gnu.org/software/libc/manual/html_node/Backtraces.html>`_.
+
+   If the system doesn't support the C-level ``backtrace()`` or ``backtrace_symbols`` functions,
```

Comment:
You can add manpage to `backtrace` using the `manpage` role.

Reply:
TIL!